design/digest.md

```diff
@@ -5,6 +5,15 @@
 
 - There are implications of the procedure Champ manages building and publishes application stacks for usage in Kabanero and the enterprise's container tagging strategy.  
 
+- The digest design fits into a series of improvements (designs) we are making to more precisely govern, visualize and manage stack lifecycles. This design relates to features for Champ which will
+
+  - Enable: Precise container governance for builds, which allows Champ and Todd fine-grained control over what specific instance of a container is being used to build new applications.  This feature can be leveraged such that Champ and Todd do not build any retired or non-approved application stack version.
```

Comment:
Clarify that today, Champ and Todd would have to agree on the stack governance policy, as there is only one location this is controlled from.  Both Todd and Champ do not have their own explicit controls.   Todd is mentioned a few more times further down as well.